### PR TITLE
Added images for Deploy using UI section

### PR DESCRIPTION
**Pull request by jay-saldanha**
Added images for Deploy using UI section. I have added 8 image files.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/683)

### CODE REVIEW

**Comment by jay-saldanha** [comment]
Thank you for suggesting these new screenshots! I have a few comments for
discussion:

   -

   For the YAML file: we should use a textual code block rather than an
   image. - ok, I will do this change
   -

   For the screenshot of the Kubeflow deployment UI, let's use just the
   part of the screenshot that shows the input fields, as the current doc
   does. That will make the screenshot smaller, make it easier to read, and
   future-proof us against changes to the help text on the left. Plus, the
   help text is out of date and missing some fields. - ok, I put the full
   image because we write in the text that there are instructions on the left,
   but we don't really show them. but if it looks too big, we can revert to
   the old image
   -

   For the three login options, I'm not sure it's a good idea to have a
   separate screenshot for each one, though I'm happy to hear arguments. :)
   The screenshots take up a lot of room, and are subject to rapid change.
   Instead, one screenshot with a clear textual guide to the options should be
   enough? - the three login options will be three different sections which
   will not be part of the original text. in the original text I will just
   mention the three options and link them to their descriptions. see below:
   -

      [Login with GCP Iap](#Login_with_GCP_IAPSection)
      -

      [Login with Username Password](#Login_with_UsernamePassword_Section)
      -

      [Setup Endpoint later](#Setup_endpoint_later_Section) . The reason is
      that there are different input requirements for all three options.
      -

   Similarly, I don't think we need separate screenshots of buttons and
   menu drop down options. Just bold text in the instructions to indicate the
   button names. - I can just point the user to the image of the form and we
   can remove these.

Overall, we'd like to keep the guide as short and concise as possible, but
without sacrificing clarity.


*Jayana Saldanha | *Technical Writer (Akorbi) | Cloud AI Platform Docs





*From: *Sarah Maddox <notifications@github.com>
*Date: *Wed, May 8, 2019 at 6:33 AM
*To: *kubeflow/website
*Cc: *Jazz4Code, Mention

*@sarahmaddox* requested changes on this pull request.

**Comment by sarahmaddox** [comment]
I'm cc-ing @ccarpentiere, because I'll be getting on a plane tomorrow for the long trip back home, arriving Saturday morning. Christa may be able to help with reviews during that time.

/cc @ccarpentiere

**Comment by sarahmaddox** [comment]
Instead of continuing with this PR, how about instead uploading the required image(s) to your PR that updates the doc. That will solve the problems you're having referencing the images in that PR. Then you can just close this PR.

**Comment by sarahmaddox** [comment]
@jay-saldanha How are you going with this PR - are you planning to keep the images in a separate branch from the related textual changes?

On the image names: capital letters are significant - a file with caps in the name is a different file from one without caps although the letters may be the same. Please would you change all caps to lower case in all the file names. For example:
Kubeflow-deployment.PNG -> kubeflow-deployment.png

And the same for the other 2 images, for consistency.

**Comment by jay-saldanha** [comment]
@sarahmaddox 

https://github.com/jay-saldanha/website/pull/2 I sent this PR and assigned it to you and Kunming 5 days ago.

This is the new PR with the changed images:
https://github.com/jay-saldanha/website/pull/2

About the images, they are opening fine now, so we should be good? Also the extension .PNG is capitalized by my computer and I can't really change it. It shouldn't matter, right?

**Comment by sarahmaddox** [comment]
> https://github.com/jay-saldanha/website/pull/2 I

Hallo @jay-saldanha that PR is on your fork of the repo. Please would you update the current PR on the kubeflow/website repo (https://github.com/kubeflow/website/pull/683) rather than a PR on your own fork.

File: content/docs/gke/deploy/deploy-ui.md [hunk 1]
```diff
@@ -6,35 +6,37 @@ weight = 3
 
 This page provides instructions for using the Kubeflow deployment web app to
 deploy Kubeflow on GCP. If you prefer to use the
-command line, see the guide to 
-[deployment using the CLI](/docs/gke/deploy/deploy-cli).
+command line, see the guide to [deployment using the CLI](/docs/gke/deploy/deploy-cli).
 
-1. Open [https://deploy.kubeflow.cloud/](https://deploy.kubeflow.cloud/#/deploy)
+<ol>
```
Comment (sarahmaddox) [inline content/docs/gke/deploy/deploy-ui.md]: @jay-saldanha Thanks for these updates! Here's a summary of the work that still needs doing to finish off this PR:

* As it currently stands, this PR changes most of the page source from Markdown to HTML. We prefer Markdown for readability and easier maintenance, though we do sometimes use HTML within a Markdown file for complex tables, adding anchor tags, or other formatting that is difficult in Markdown. TODO: Revert the changes where HTML formatting is not required.

* There are also some formatting/syntax issues on the page - couple of review comments from the related PR https://github.com/kubeflow/website/pull/685 (now closed) haven't made it into this PR.

* As you mentioned, there are still some comments in the HTML that need resolving.

**Comment by sarahmaddox** [comment]
jay-saldanha and I have chatted about this PR and decided that I'll take this work over, making use of the content from the two PRs mentioned in the comments. Thanks so much for your contributions @jay-saldanha - the material you've added will be very useful in creating a followup PR!

/close

**Pull request closed without merging**